src/components/ChartSetting.jsx

```diff
@@ -177,23 +188,25 @@ const ChartSetting = ({
 };
 
 export default connect(({ chartSetting: s, chart: c }) => ({
-    ChartSettingMenu: s.ChartSettingMenu,
-    menuOpen: s.menu.dialog.open,
-    selectedLanguage: s.language,
-    languages: s.languages,
-    position: s.position,
-    setPosition: s.setPosition,
-    setView: s.setView,
-    view: s.view,
-    setLanguage: s.setLanguage,
-    theme: s.theme,
-    setTheme: s.setTheme,
-    countdown: s.countdown,
-    showCountdown: s.showCountdown,
-    closeMenu: s.menu.onTitleClick,
-    assetInformation: s.assetInformation,
+    assetInformation   : s.assetInformation,
+    ChartSettingMenu   : s.ChartSettingMenu,
+    closeMenu          : s.menu.onTitleClick,
+    countdown          : s.countdown,
+    historical         : s.historical,
+    isMobile           : c.isMobile,
+    isAutoScale        : s.isAutoScale,
+    languages          : s.languages,
+    menuOpen           : s.menu.dialog.open,
+    position           : s.position,
+    selectedLanguage   : s.language,
     setAssetInformation: s.setAssetInformation,
-    historical: s.historical,
-    setHistorical: s.setHistorical,
-    isMobile: c.isMobile,
+    setAutoScale       : s.setAutoScale,
+    setHistorical      : s.setHistorical,
+    setLanguage        : s.setLanguage,
+    setPosition        : s.setPosition,
+    setTheme           : s.setTheme,
+    setView            : s.setView,
+    showCountdown      : s.showCountdown,
+    theme              : s.theme,
+    view               : s.view,
 }))(ChartSetting);
```

Comment:
@morteza-binary could you pls do refactoring and adding new codes in separate comments , it's hard to review cause its hard to find new codes among last refactored codes.

Reply:
@MahboobehMohammadi Actually it'd be better to apply the style changes in ongoing PRs instead of postponing everything to one day in the future.

Reply:
@morteza-binary yes same PRs, but should be in different comments cause otherwise reviewing will be impossible.